src/objectExplorer/objectExplorerUtils.ts

```diff
@@ -54,4 +54,8 @@ export class ObjectExplorerUtils {
         }
         return LocalizedConstants.defaultDatabaseLabel;
     }
+
+    public static isFirewallError(errorMessage: string): boolean {
+        return errorMessage.includes(Constants.firewallErrorMessage);
+    }
```

Comment:
This is really starting to get to be a huge annoyance that we don't return the error code. Have you looked into adding the error code to the session response? That would be way more preferable than doing stuff like this...for example what happens if the error message we get back is localized? The message is from the engine right? So it might not be in English in that case...

Reply:
That is something I planned to do right after making this PR. It would need some work on the tools service side so I don't want to do it now, but after this release I would definitely add error codes.